0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -504,14 +508,48 @@
         )
 
     image_corr_df = (
-        pd.concat(image_corr_list).drop_duplicates(subset="site_location").reset_index()
+        pd.concat(image_corr_list).drop_duplicates(subset="site").reset_index()
+    )
+
+    image_corr_df["min_corr_value"] = (
```

Comment:
is this creating a new variable? For consistency, lets use the notation:

```python
image_corr_df = image_corr_df.assign(min_corr_value = ...)
```

Also, the new column is "min_corr_value" but we're taking the max? Am i reading this right?

Reply:
That's a mistake. Fixed max => min.